gdp-src-build/conf/templates/local.inc

```diff
@@ -37,3 +37,5 @@ CONF_VERSION = "1"
 INHERIT += "testexport"
 TEST_EXPORT_ONLY = "1"
 TEST_SUITES = "ping ssh scp date dmesg connman systemd gdp devpackages"
+
+CUSTOM_IMAGE_FSTYPE ??= "tar.gz"
```

Comment:
Please add a comment above this.  The commit comment explains but I think it is worthwhile to have it here.  For example:

\# This will be assigned to IMAGE_FSTYPE later.  The default is used if the type not defined in machine-specific templates.  At least one image type is required or the build will fail.

Reply:
Good point